### ISSUE TITLE
Importer creates animation tracks even for non-animated vertex groups

### ISSUE THREAD

**Issue by tagcup** [godotengine/godot#6595]
**Operating system or device - Godot version:**
Debian GNU/Linux

**Issue description** (what happened, and what was expected):
I have a model created in Blender with Actions. Some actions don't involve certain bones (clothing); this is on purpose, they will be modified via GDScript dynamically in the game based on kinematics.

However, Godot importer creates a track in all animations for such bones nevertheless (with two keys: one at the beginning and one at the end of the animation, both in the default position of the bone). This interferes with the in-game kinematic based control. 

A possible workaround is to remove all such tracks manually by hand or using an import script. 

**Steps to reproduce:**
Create a model in Blender and create an action, but don't create tracks for all bones. Export using Better Collada and use Godot's scene importer.

**Link to minimal example project** (optional but very welcome):

**Comment by reduz** [comment]
Yes this is kinda impossible to know from Godot, specially if the bone is
posed

**Comment by tagcup** [comment]
In my case, those additional bones aren't posed (otherwise it should show up in the dope sheet I guess); I just checked the keys in the additional tracks and they all contain:
```
loc: (0,0,0)
rotate: (0,0,0,1)
scale: (1,1,1)
easing: 1
```

So just to confirm whether I'm understanding what're saying correctly, are you saying that the Better Collada Export is exporting this additional information (that wasn't there in the first place), and Godot can't tell whether these values are actual or made-up data after the fact by just looking at the dae file?

**Comment by Noshyaar** [comment]
First of all thank you for your report and sorry for the delay.

We released Godot 3.0 in January 2018 after 18 months of work, fixing many old issues either directly, or by obsoleting/replacing the features they were referring to.

We still have hundreds of issues whose relevance/reproducibility needs to be checked against the [current stable version](https://godotengine.org/download), and that's where you can help us.
**Could you check if the issue that you described initially is still relevant/reproducible in Godot 3.0 or any newer version, and comment about it here?**

For bug reports, please also make sure that the issue contains detailed steps to reproduce the bug and, if possible, a zipped project that can be used to reproduce it right away. This greatly speeds up debugging and bugfixing tasks for our contributors.

Our Bugsquad will review this issue more in-depth in 15 days, and potentially close it if its relevance could not be confirmed.

Thanks in advance.

*Note: This message is being copy-pasted to many "stale" issues (90+ days without activity). It might happen that it is not meaningful for this specific issue or appears oblivious of the issue's context, if so please comment to notify the Bugsquad about it.*

**Comment by organicpencil** [comment]
I think this can be closed. It's an exporter thing. See https://github.com/godotengine/collada-exporter/issues/37
